spec/facility_factory_spec.rb

```diff
@@ -0,0 +1,92 @@
+require 'spec_helper'
+
+
+RSpec.describe FacilityFactory do
+    before(:each) do
+        @facilities = FacilityFactory.new
+        DmvDataService.new.co_dmv_office_locations
+        @co_source = DmvDataService.new.co_dmv_office_locations 
+        @ny_source = DmvDataService.new.ny_dmv_office_locations
+        @mis_source = DmvDataService.new.mo_dmv_office_locations
+    end  
+
+    describe '#colorado location attributes' do
+    it 'can count the number of locations in the array' do
+       
+        expect(@facilities.co_facilities(@co_source).count).to eq(5)
```

Comment:
Validate the data on the first element of this collection! Ah, you are doing this below actually - nice work!